Description: 
 -Add iddef TIOCINQ = FIONREAD
 jamvm/src/classlib/openjdk/jvm.c
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2011-06-28
Forwarded: no
Index: jamvm/jamvm/src/classlib/openjdk/jvm.c
===================================================================
--- jamvm/jamvm/src/classlib/openjdk/jvm.c.orig
+++ jamvm/jamvm/src/classlib/openjdk/jvm.c
@@ -63,6 +63,10 @@
 #define have_monotonic_clock FALSE
 #endif
 
+#ifndef TIOCINQ
+#define TIOCINQ FIONREAD
+#endif
+
 static Class *cloneable_class, *constant_pool_class;
 static Class *exception_class, *runtime_excp_class;
 
